data/presets/amenity/stripclub.json

```diff
@@ -0,0 +1,40 @@
+{
+    "icon": "fas-place-of-worship",
+    "fields": [
+        "name",
+        "operator",
+        "address",
+        "building_area_yes",
+        "opening_hours",
+        "min_age",
+        "smoking"
+    ],
+    "moreFields": [
+        "air_conditioning",
+        "email",
+        "fax",
+        "gnis/feature_id-US",
+        "level",
+        "payment_multi",
+        "opening_hours/covid19",
+        "phone",
+        "ref/FR/siret-FR",
+        "wheelchair",
+        "fee"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "amenity": "stripclub"
+    },
+    "terms": [
+        "lap dance",
+        "adult*",
+        "gentlemen*",
+        "cabaret",
+        "strip*"
```

Comment:
what's the purpose of the asterisks `*` here?

Reply:
There are lots of things that begin with gentlemen or adult, and many of them are strip clubs? I assumed from nightclub.json that the asterisks were required. https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/amenity/nightclub.json#L33

Reply:
I see.  :thinking: However, I cannot find any reference in the code where anything special is done with the `*` character in preset terms. I would assume that in the early prototype versions of iD, this was supposed to indicate that iD should match all search input starting with the respective term (e.g. that searching for `discoteque` would find the `amenity=nightclub` preset which was `disco*` as a term). However, as far as I can see, this was never implemented.  @bhousel does that sound right, or is some other reason for the `*` in the terms of some presets?

Reply:
> However, as far as I can see, this was never implemented. @bhousel does that sound right, or is some other reason for the * in the terms of some presets?

I'm not sure , using `*` as a wildcard might have been supported a very long time ago, but I don't think the `*`'s do anything now.